### DISCUSSION TITLE
Visit island

### DISCUSSION THREAD

**Discussion by RickyLaChow** [BG-Software-LLC/SuperiorSkyblock2#153] [Pending Feature Requests]
Add the possibility for visit an island without the player need set a visitators spawn
Or a toggle option for have visitators or not

Or anyway add /island setvisitators spawn or something else

[the remark this answers is not in the document]
**Reply by RickyLaChow** [reply]
I set a warp, public warp, but when a player do /island visit name say don't have any spawn for visitators

[the remark this answers is not in the document]
**Reply by OmerBenGera** [reply]
> 
> 
> I set a warp, public warp, but when a player do /island visit name say don't have any spawn for visitators

It's a warp - they need to use /is warp for that.

[the remark this answers is not in the document]
**Reply by RickyLaChow** [reply]
Ok, but there are a command /island visit (name)

You can add when i type the command i will spawn in the island? If is not in private Status?

[the remark this answers is not in the document]
**Reply by RickyLaChow** [reply]
I think is very easy for people and cool

[the remark this answers is not in the document]
**Reply by OmerBenGera** [reply]
> 
> 
> Ok, but there are a command /island visit (name)
> 
> You can add when i type the command i will spawn in the island? If is not in private Status?

The /is visit is for the welcome sign, similar to ASkyblock. You are mixing here two different systems.

[the remark this answers is not in the document]
**Reply by RickyLaChow** [reply]
Okok, my proposal is a command to visit player island without setwarps or anyother things

[the remark this answers is not in the document]
**Reply by Skizzles** [reply]
> Okok, my proposal is a command to visit player island without setwarps or anyother things

For player usage or admin usage? 
As there is /is admin tp (name) which will send you to the island.

[the remark this answers is not in the document]
**Reply by RickyLaChow** [reply]
> > Okok, my proposal is a command to visit player island without setwarps or anyother things
> 
> For player usage or admin usage? As there is /is admin tp (name) which will send you to the island.

The propouse was for player
